scripts/Plotter.py

```diff
@@ -0,0 +1,241 @@
+#!/usr/bin/env python3
+
+# SPDX-FileCopyrightText: Intel Corporation
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+import json
+import os
+import re
+from dataclasses import dataclass
+
+import click
+import matplotlib.pyplot as plt
+
+
+@dataclass
+class BenchResults:
+    copy: float
+    scale: float
+    add: float
+    triad: float
+
+
+class Plotter:
+    def __init__(self) -> None:
+        self.plots_path_shp = "."
+        self.plots_path_mhp = "."
+        self.bench_path_shp = "."
+        self.bench_path_mhp = "."
+        self.create_directories()
+
+    def create_directories(self):
+        if not os.path.exists(self.plots_path_shp):
```

Comment:
See https://stackoverflow.com/a/41959938/2525421

Reply:
Done but it for sure has to be changed because of Lukasz PR with packaging and new directories